actions-cleanup-openstack-vms/action.yaml

```diff
@@ -0,0 +1,85 @@
+name: 'Clean up OpenStack vms'
+description: 'this action is to clean up vms in openstack'
+
+
+inputs:
+  DELTA:
+    description: the default age of the VM
+    default: 6
```

Comment:
6 what? hours? we need to say it somewhere

Reply:
ok